### PR TITLE
Replace Outbrain widget with native component

### PR DESCRIPTION
**Pull request by alexanderedge**
This is the template part of [GLA-1238](https://theguardian.atlassian.net/browse/GLA-1238).

I've pretty much copied the implementation of related content and removed everything related to the existing Outbrain JS widget.

I'm assuming that once this is merged into `master` we *cannot* do any other template changes until https://github.com/guardian/ios-live/tree/gla-1238-outbrain-sdk is also merged into master. Is that correct?

### Testing
Please correct my if I'm wrong in any of the following:

1. Check out this branch locally
2. Build the templates using `npm run build`
3. Check out the [iOS branch](https://github.com/guardian/ios-live/tree/gla-1238-outbrain-sdk) locally
4. Edit `package.json`, assuming that you've checked out the templates in the same parent directory as the iOS project:

```json
"dependencies": {
    "@guardian/mobile-apps-article-templates": "file:../mobile-apps-article-templates"
}
```
5. Run the iOS app *without a subscription*
6. You should see Outbrain recommendations at the bottom of articles.

### CODE REVIEW

**Comment by webb04** [comment]
Yes we should only merge these changes once the iOS changes are in master.

**Comment by alexanderedge** [comment]
Thanks, Jamie, I'll check the tests. I think open a separate PR for changes to comments and that can get merged before this if ready.

**Comment by alexanderedge** [comment]
Also, I have a bunch of changes to make design wise (iPad also needs 4-across). I'll be in touch when it's ready for another look.

File: ArticleTemplates/assets/js/bootstraps/article.js [hunk 1]
```diff
@@ -1,17 +1,11 @@
...
-import { init as initOutbrain } from 'modules/outbrain';
 import { init as quizInit } from 'modules/quiz';
 import { init as immersiveInit } from 'modules/immersive';
 import { init as creativeInjectorInit } from 'modules/creativeInjector';
 import { init as messengerInit } from 'modules/messenger';
 import resizeInit from 'modules/messenger/resize';
 
-function setupGlobals() {
-    window.articleOutbrainInserter = initOutbrain;
-    window.applyNativeFunctionCall('articleOutbrainInserter');
```
Comment (webb04) [inline ArticleTemplates/assets/js/bootstraps/article.js]: There's one more use of this function in `mobile-apps-article-templates/ArticleTemplates/assets/js/boot.js`

File: ArticleTemplates/outbrainTemplate.html [hunk 2]
```diff
@@ -1,5 +1 @@
-<div class="container container__outbrain" id="outbrain">
```
Comment (webb04) [inline ArticleTemplates/outbrainTemplate.html]: There's a few CSS styles targeting this selector that can probably be removed `.container__outbrain`

**Pull request merged**